Introduction.md

```diff
@@ -1738,7 +1738,7 @@ FIXME: Write how to investigate a test failure.
 ## Debugging Layout Tests in Xcode
 
 The easiest way to debug a layout test is with WebKitTestRunner or DumpRenderTree.
-In Product > Scheme, select “All Source”.
+In Product > Scheme, select “Everything up to WebKit + Tools”.
```

Comment:
This is intentionally not just a rename. The "All Source" scheme wouldn't build WebKitTestRunner or DumpRenderTree, so I don't see how it could be used to run them unless you had already built them first.

Reply:
That wasn't necessary in most cases, as `run-webkit-tests` compiles them. Could cause trouble when running the tools directly, without the script.

Reply:
From the screenshot included here, it looks like DumpRenderTree used to be part of "All Sources":

![](https://github.com/WebKit/WebKit/raw/main/resources/xcode-build-settings-for-run.png)

IIUC, these directions aren't telling the reader to use `run-webkit-tests` at all. Regardless, it would be good to go through this workflow again, verify it works, and take new screenshots.